recipes/loos/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "loos" %}
+{% set version = "4.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GrossfieldLab/loos/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a0c4a0c9ed367a1b8cb6ed22fe08e24a8b4a78785d072e8be96013c9839f6671
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - swig
+  host:
+    - python
+    - pip
+    - numpy
+    - boost-cpp
+    - openblas
```

See https://conda-forge.org/docs/maintainer/knowledge_base.html#blas
```suggestion
    - libblas
```